*** tests/A-01-03-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-01-03-M-ok.outhyp	2019-11-14 13:41:28.498639544 +0000
***************
*** 12,19 ****
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: 0 - 50 de  50 - Livro - Bairro Alto e o Budismo Zen - 25 - Ficção - Zun Tse Fonseca - 1234567891
! Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
--- 12,18 ----
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
