*** tests/A-02-03-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-02-03-M-ok.outhyp	2019-11-14 15:58:28.931880385 +0000
***************
*** 12,20 ****
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: 0 - 10 de  10 - DVD - Era uma vez na Amadora - 20 - Ficção - Fernando Fonseca - 200505550
! 1 - 10 de  10 - DVD - 48 Horas para o Exame - 12 - Ficção - Orlando Fonseca - 200505553
! 2 - 5 de  5 - DVD - Lumiar Selvagem - 20 - Ficção - Pedro Fonseca - 200505551
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 12,19 ----
  2 - Mostrar todas as obras
  3 - Efectuar pesquisa de termos
  0 - Sair
! Escolha uma opção: 0 - 10 of 10 - DVD - Era uma vez na Amadora - 20 - FICTION - Fernando Fonseca - 200505550
! 1 - 10 of 10 - DVD - 48 Horas para o Exame - 12 - FICTION - Orlando Fonseca - 200505553
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
