*** tests/A-05-02-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-05-02-M-ok.outhyp	2019-11-14 14:24:45.058844308 +0000
***************
*** 14,23 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
! 1 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
! 2 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 14,20 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
