*** tests/A-06-01-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-06-01-M-ok.outhyp	2019-11-13 13:27:57.346394470 +0000
***************
*** 1,20 ****
- Menu Principal
- 1 - Abrir dados
- 2 - Guardar dados
- 3 - Ver data
- 4 - Avançar data
- 5 - Menu de gestão de utentes
- 6 - Menu de gestão de obras
- 7 - Menu de gestão de requisições
- 0 - Sair
- Escolha uma opção: Data actual: 0
- Menu Principal
- 1 - Abrir dados
- 2 - Guardar dados
- 3 - Ver data
- 4 - Avançar data
- 5 - Menu de gestão de utentes
- 6 - Menu de gestão de obras
- 7 - Menu de gestão de requisições
- 0 - Sair
- Escolha uma opção: 
\ No newline at end of file
--- 0 ----
